runtime/parachains/src/inclusion.rs

```diff
@@ -168,7 +173,9 @@ impl<T: Trait> Module<T> {
 
 		let mut assigned_paras_record: Vec<_> = (0..n_bits)
 			.map(|bit_index| core_lookup(CoreIndex::from(bit_index as u32)))
-			.map(|core_para| core_para.map(|p| (p, PendingAvailability::<T>::get(&p))))
+			.map(|core_para| core_para.map(|p| { 
+				(p, PendingAvailability::<T>::get(&p), PendingAvailabilityCommitments::get(&p))
```

Comment:
Most of the goal here is actually to avoid looking up the commitments eagerly as they can be heavy. It would be best to look them up only lazily when calling `enact_candidate`.